modules/programs/neovim.nix

```diff
@@ -43,7 +43,7 @@ let
 
   # A function to get the configuration string (if any) from an element of 'plugins'
   pluginConfig = p:
-    if builtins.hasAttr "plugin" p && builtins.hasAttr "config" p then ''
+    if builtins.hasAttr "plugin" p && p.config or "" != "" then ''
```

Comment:
Aren't both versions equivalent? I find more readable the `hasAttr` version

Reply:
An for the shortest version: `p ? plugin && p ? config`